### ISSUE TITLE
Dropdown, MultiSelect, AutoComplete, CascadeSelect: aria-label missing from inner ul element

### ISSUE THREAD

**Issue by akashuwtw** [primefaces/primeng#14790]
### Describe the bug

We are trying to set `aria-label` for `p-dropdown` which is getting successfully applied to the main `p-dropdown` component, but is missing from it's inner `ul` element.

We are following the steps as per documented [here](https://primeng.org/dropdown#accessibility).
 

### Environment

OS: Windows 10

### Angular version

16.2.7

### PrimeNG version

16.5.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.16.0

### Browser(s)

Chrome 121.0.6167.161, Edge 121.0.2277.112

### Steps to reproduce the behavior

`<p-dropdown data-test="type" [options]="typeOptions" formControlName="type" placeholder="Select type"></p-dropdown>`
`<p-dropdown data-test="type" [options]="typeOptions" formControlName="type" ariaLabel="Select type"></p-dropdown>`
`<p-dropdown data-test="type" [options]="typeOptions" formControlName="type" aria-label="Select type"></p-dropdown>`

### Expected behavior

aria-label should be applied to inner ul element

**Comment by cetincakiroglu** [comment]
Hi,

ariaLabel and ariaLabelledBy are given to focusable input elements. The test we performed using Axe DevTools shows that there is no issue as you mentioned.

**correct use:** 

```
<span id="dd1">Options</span>
<p-dropdown ariaLabel="myAriaLabel"/>
```

This is a DOM node from the basic example in primeng.org. As you see here list item has aria-label and it's generated by option label. The ﻿`ariaLabel` prop you set in the dropdown is for the focusable element that replaces the ﻿<input> in the dropdown, not for the list items.

```
<li role="option" pripple="" class="p-ripple p-element p-dropdown-item p-focus" id="pn_id_7_0" ng-reflect-ng-style="[object Object]" ng-reflect-ng-class="[object Object]" aria-label="New York" aria-setsize="5" aria-posinset="1" aria-selected="false" data-p-focused="true" data-p-highlight="false" data-p-disabled="false"><span class="ng-star-inserted">New York</span><!--bindings={
  "ng-reflect-ng-if": "true"
}--><!--bindings={
  "ng-reflect-ng-template-outlet-context": "[object Object]"
}--><span class="p-ink" aria-hidden="true" role="presentation"></span></li>
```

**Comment by akashuwtw** [comment]
Hi,

We are performing [cypress axe](https://github.com/component-driven/cypress-axe) tests to check for any **a11y** violations, and cypress has reported this violation for `ul `element under `p-dropdown`

![image](https://github.com/primefaces/primeng/assets/152413670/3388d927-d067-4495-9073-f02f6c1fb439)

`<ul role="listbox" class="p-dropdown-items ng-star-inserted" id="pn_id_5_list">`
https://dequeuniversity.com/rules/axe/4.8/aria-input-field-name?application=axeAPI

Since, the `ul` element has `listbox` role, it become an input field and hence should have an `aria-label`.

The `li` elements under `ul` element have `aria-label` applied. No issues there.
![image](https://github.com/primefaces/primeng/assets/152413670/f7d23272-946e-48d6-b999-67551c27a1aa)

Regards,
Akash

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the clarification. In this case, do you need a default assigned ﻿aria-label value or do you want to send the ﻿aria-label via an input?

**Comment by akashuwtw** [comment]
Hi,

The default aria-label would be fine for us.

Regards,
Akash

**Comment by cetincakiroglu** [comment]
Hi,

Thanks a lot for the feedback, we'll add an aria label to the element.
